atox/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,120 @@
+        <item>Keinen</item>
+        <item>HTTP</item>
+        <item>SOCKS5</item>
+    </string-array>
+    <string name="version_display">aTox v%1$s - %2$d</string>
+    <string name="send_as_action">Als Aktion senden</string>
+    <string name="or_import_existing_save">Oder Importiere ein bestehendes Tox Profil</string>
+    <string name="profile_information">Profil-Information</string>
+    <string name="profile_settings">Profil-Einstellungen</string>
+    <string name="profile_nickname">Ändere deinen Namen</string>
+    <string name="profile_status_message">Ändere deine Statusmeldung</string>
+    <string name="profile_status">Ändere deinen Status</string>
+    <string name="profile">Profil</string>
+    <string name="tox_id_error_already_exists">Einen Kontakt mit dieser Tox-ID gibt es bereits</string>
+    <string name="tox_service_running">aTox läuft</string>
+    <string name="pref_heading_network">Netzwerk</string>
+    <string name="pref_udp_enabled">UDP aktiviert</string>
+    <string name="pref_heading_general">Allgemein</string>
+    <string name="pref_run_at_startup">Beim Hochfahren ausführen</string>
+    <string name="friend_request">Freundschaftsanfrage</string>
+
+    <string name="status_available">Anwesend</string>
+    <string name="status_away">Abwesend</string>
+    <string name="status_busy">Beschäftigt</string>
+    <string-array name="user_statuses">
+        <item>@string/status_available</item>
+        <item>@string/status_away</item>
+        <item>@string/status_busy</item>
+    </string-array>
+    <string name="from">Von</string>
+
+    <!-- Status dialog -->
+    <string name="status_dialog_edit_user_status">Status ändern</string>
+    <string name="mimetype_handler_not_found">Es konnte keine Anwendung gefunden werden, welche diesen Dateityp öffnen kann %1$s</string>
+    <string name="attach_file">Datei anhängen</string>
+    <string name="delete">Löschen</string>
+    <string name="notification_file_transfer">Eingehende Datenübertragung \"%1$s\"</string>
+    <string name="cancelled">Abgebrochen</string>
+    <string name="completed">Abgeschlossen</string>
+    <string name="save_as">Speichern unter…</string>
+    <string name="export_file_success">Datei erfolgreich gesendet</string>
+    <string name="export_file_failure">Etwas ist beim Export schief gegangen. Pre-0.5.1 Übertragung wurde standardmäßig exportiert.</string>
+    <string name="draft_message">Entwurf: %1$s</string>
+    <string name="pref_heading_address">Adresse</string>
+    <string name="pref_heading_port">Port</string>
+    <string name="pref_heading_proxy">Proxy</string>
+    <string name="warn_proxy_broken">Falscher Proxy konfiguriert. Vor Verwendung der App beheben oder deaktivieren.</string>
+    <string name="proxy_not_found">Proxy nicht gefunden</string>
+    <string name="bad_type">Defekter Typ</string>
+    <string name="bad_host">Defekter Host</string>
+    <string name="bad_port">Defekter Port</string>
```

Comment:
"Defective" sounds like something is broken. Probably "invalid" is better, so: "Ungültiger" for all three strings.